doc/settings.md

```diff
@@ -70,3 +70,13 @@ Newer versions of [nixpkgs] provide `buildFromSdist` to build your package from
 > If you encounter issues with `buildFromSdist` you can disable it by setting `settings.<name>.buildFromSdist` to `true`.
 
 [nixpkgs]: https://nixos.asia/en/nixpkgs
+
+### `stan`
+
+Run **ST**atic **AN**alysis on the package using [stan] and generate an HTML report. The report is created in the `/nix/store` path alongside your package outputs.
+
+> [!note] stan configuration  
+> This setting looks for a `.stan.toml` file in the root of the package source. See a sample [.stan.toml] configuration for reference.
```

Comment:
A bit of hardcoding here. I guess we could eliminate this by making the stan setting a submodule (providing an option to configure custom configuration path) instead of just a boolean, but I doubt that’s necessary.

Reply:
I followed these instructions and added `.stan.toml` to the root:

https://github.com/juspay/vira/commit/53a42bf8a6bf627e9690ea402ce67d2036b59295

However, it doesn't seem to be used, because the generated report still doesn't exclude the StrictData suggestion:

![image](https://github.com/user-attachments/assets/3d681484-e058-44ac-9a03-183a844f1a71)

It'd be good to demonstrate this in a sample repo, like https://github.com/srid/haskell-template - along with `.stan.toml`.

Reply:
cf. https://github.com/srid/haskell-template/pull/147